sdk/nodejs/cmd/run-plugin/index.ts

```diff
@@ -0,0 +1,127 @@
+// Copyright 2016-2024, Pulumi Corporation.
```

Comment:
```suggestion
// Copyright 2024, Pulumi Corporation.
```

Reply:
I'm not sure here what the right thing to do is actually.  I left the 2016 intentionally, since the file I copied this from had it, so I thought it still needs the old copy right associated with it. Happy to change though.